package.json

```diff
@@ -9,17 +9,20 @@
   "scripts": {
     "start": "npm run build && node .",
     "build": "lb-tsc && lb-tslint",
-    "test": "npm run build && mocha --recursive ./dist/test/**/*.test.js"
+    "test": "npm run build && mocha --recursive \"./dist/test/**/*.test.js\""
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/strongloop/loopback4-example-getting-started.git"
+    "url":
+      "git+https://github.com/strongloop/loopback4-example-getting-started.git"
```

Comment:
FWIW, we are not (or should not be) applying prettier formatting to `package.json`, because `npm` will reformat the file back to `JSON.stringify` format whenever we change it, typically via `npm install`. Could you please revert these white-space only changes?

I am not sure if `package.json` is listed in `.prettierignore`, feel free to add it there if it isn't. Here is what we did in loopback4-extension-starter (the trick works well with VS Code prettier plugin): 

https://github.com/strongloop/loopback4-extension-starter/blob/3a491d40e7d58ad177484de459050ba127215af7/.prettierignore#L4

```
*.json
```